posts/using-napari-for-cosmx-data/index.qmd

```diff
@@ -0,0 +1,525 @@
+[series topics](https://nanostring-biostats.github.io/CosMx-Analysis-Scratch-Space/about.html) for what's coming ahead!
+
+
+# Using the `napari-cosmx` plugin 
+
+
+::: {.callout-note}
+While napari and the `napari-cosmx` can be installed on many systems, CosMx data
+can be quite large. A slide with a large number of FOVs may very well exceed the
+capabilities of a standard laptop.
+:::
+
+
+## Installing `napari-cosmx` {#sec-installing}
+
+The specific download instructions depend on your operating system (see tabs below)
+but the general procedure is the same: 1) installing napari 0.4.17 and 2) installing the `napari-cosmx` plugin
+
+::: {.callout-note}
+The `napari-cosmx` plugin was developed with napari 0.4.17. There are some breaking
+changes that we have noticed if using the plugin with newer version of napari (_e.g._,
+0.4.18). At the time of writing this post, please make sure to install version 0.4.17. 
+
+:::
+
+::: {.panel-tabset}
+
+### Windows Install
+
+::: {.callout-note}
+Depending on your browser and security settings you may get warnings when downloading or running some of the links below.
+:::
+
+#### Part 1: Installing Napari
+
+:::: {.columns}
+
+::: {.column width="45%"}
+
+The napari project contains platform-specific bundled apps for each release that
+don’t require you to first install a Python environment. You simply run the installer
+and a link will be added to your Start Menu as with a typical app installation. 
+The napari-CosMx plugin currently expects napari 0.4.17.
+
+[Click to download the Windows Installer](https://github.com/napari/napari/releases/download/v0.4.17/napari-0.4.17-Windows-x86_64.exe)
+
+:::
+
+::: {.column width="5%"}
+<!-- empty column to create gap -->
+:::
+
+::: {.column width="40%"}
+
+```{r}
+#| eval: true
+#| echo: false
+#| label: "fig-windows-installer"
+#| fig-cap: "The Windows Installer."
+
+knitr::include_graphics("./figures/windows_installer.png")
+```
+
+:::
+
+#### Part 2: Installing `napari-cosmx`
+
+1. Download the whl file (**need a location for this**). 
+2. Go to the folder where the `whl` file is in your File Explorer and choose 
+`Copy Path`.
+2. Now launch napari from the Start menu in Windows. 
+
+> Pro tip: If launching for the first time, the application may take a moment to appear. Avoid launching multiple instances. 
+
+3. In napari, open up the `>_` button that is located on the bottom left (see @fig-windows-plugin-command for example). 
+4. Type `pip install ` into the console (_i.e._, with a single space after the word 'install').
+5. Paste the location of the `whl` file
+6. Press enter to execute
+7. You should receive a message in the console that several packages were successfully
+installed including the `napari-cosmx` package. 
+8. **Close and re-start napari**
+
+```{r}
+#| eval: true
+#| echo: false
+#| label: "fig-windows-plugin-command"
+#| fig-cap: "Example showing how to install the `napari-cosmx` file. Your file name and path will look different."
```

Comment:
Can the text in figure be made any bigger? Hard to read.
Also add to legend: 'Console open button is circled in yellow and path is underlined.'

Reply:
Figure redone. 2c07f5b40f6bc1f27aaa6391ed017a61ec807857